README.md

```diff
@@ -1,47 +1,97 @@
-# Express API Starter
-
-Includes API Server utilities:
-
-* [morgan](https://www.npmjs.com/package/morgan)
-    * HTTP request logger middleware for node.js
-* [helmet](https://www.npmjs.com/package/helmet)
-    * Helmet helps you secure your Express apps by setting various HTTP headers. It's not a silver bullet, but it can
-      help!
-* [dotenv](https://www.npmjs.com/package/dotenv)
-    * Dotenv is a zero-dependency module that loads environment variables from a `.env` file into `process.env`
-
-Development utilities:
-
-* [nodemon](https://www.npmjs.com/package/nodemon)
-    * nodemon is a tool that helps develop node.js based applications by automatically restarting the node application
-      when file changes in the directory are detected.
-* [eslint](https://www.npmjs.com/package/eslint)
-    * ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code.
-* [mocha](https://www.npmjs.com/package/mocha)
-    * ☕️ Simple, flexible, fun JavaScript test framework for Node.js & The Browser ☕️
-* [supertest](https://www.npmjs.com/package/supertest)
-    * HTTP assertions made easy via superagent.
+# Express AEViso API
+
+Team:  
+ -Aeryle: https://github.com/Aeryle  
+ -Chabelle78: https://github.com/Chabelle78  
+ -ThomasBarrial: https://github.com/ThomasBarrial  
+ -MatthiasWanner: https://github.com/MatthiasWanner  
+ -Benwade-progldc: https://github.com/Benwade-prog
+
+API REST made for AEVISO client project: https://github.com/WildCodeSchool/btz-0321-aeviso-client.  
+With jwt authentication.
 
 ## Setup
```

Comment:
Maybe also add a "Prerequisites" category with different links to downloads (Node, Docker, etc) ?